sechub-integrationtest/pds/product-scripts/integrationtest-codescan.sh

```diff
@@ -67,10 +67,31 @@ if [[ ! -f "${PDS_JOB_RESULT_FILE}" ]]; then
     echo "${PDS_JOB_RESULT_FILE} was missing - created empty file"
 fi
 
+# *******************************************
+#        Post processing of result file
+# *******************************************
+# 
+# In the next lines we post process the already existing 
+# result file:  It is either a merged file which consists
+# of the uploaded sources or a file which contains 
+# just a placeholder inside.
+#
+# To have it processed correctly from SecHub in integration
+# test mode (by a special integrationtest importer) we
+# have to add a first line as a marker. The second line
+# is a generic information for variant names etc. which
+# can be tested in unit tests as well.
+
+mv "${PDS_JOB_RESULT_FILE}" "${PDS_JOB_RESULT_FILE}_tmp"
+ 
 echo "#PDS_INTTEST_PRODUCT_CODESCAN
-info:pds.test.key.variantname as PDS_TEST_KEY_VARIANTNAME=$PDS_TEST_KEY_VARIANTNAME,product1.level as PRODUCT1_LEVEL=$PRODUCT1_LEVEL
-$(cat ${PDS_JOB_RESULT_FILE})" > "${PDS_JOB_RESULT_FILE}"
+info:pds.test.key.variantname as PDS_TEST_KEY_VARIANTNAME=$PDS_TEST_KEY_VARIANTNAME,product1.level as PRODUCT1_LEVEL=$PRODUCT1_LEVEL" > "${PDS_JOB_RESULT_FILE}"
```

Comment:
Please add a comment here explaining that the marker `info:pds.test.key.variantname` is added at this point of the execution. The marker is used  by the special integrationtest importer to recognize that it is an integration test result.